client/src/main/java/com/paypal/selion/internal/platform/grid/LocalNode.java

```diff
@@ -110,7 +110,7 @@ public void shutdown() {
         if (SystemUtils.IS_OS_WINDOWS) {
             if (!checkForPresenceOf(ConfigProperty.SELENIUM_IEDRIVER_PATH,
                     SeLionConstants.WEBDRIVER_IE_DRIVER_PROPERTY, SeLionConstants.IE_DRIVER)) {
-                Config.setConfigProperty(ConfigProperty.SELENIUM_IEDRIVER_PATH, SeLionConstants.SELION_HOME_DIR
+                Config.setConfigProperty(ConfigProperty.SELENIUM_IEDRIVER_PATH, SeLionConstants.selionHomeDir
```

Comment:
I'm not a fan of the codacy suggestion and change here. I think we make an exception for this constant.